de.metas.printing/de.metas.printing.base/src/main/java/de/metas/printing/api/impl/PrintJobBL.java

```diff
@@ -102,6 +121,8 @@ public int createPrintJobs(final IPrintingQueueSource source,
 		final PrintingQueueProcessingInfo printingQueueProcessingInfo = source.getProcessingInfo();
 
 		int printJobCount = 0;
+		final List<I_C_Print_Job_Instructions> pjis = new ArrayList<I_C_Print_Job_Instructions>();
```

Comment:
please add a comment here, and/or change the var name to indicate wtf it is about (afaiu it collects C_Print_Job_Instructions for the virtual PDF printer)

i also suggest to just collect them in a dedicated little method and at any rate try to *reduce* the nesting depth in this method, rather than incresing it..